src/System.Buffers.Experimental/System/Buffers/Text/BufferWriter.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+
+namespace System.Buffers.Text
+{
+    public ref partial struct BufferWriter
+    {
+        Span<byte> _buffer;
+        int _written;
+
+        public ReadOnlySpan<byte> NewLine;
+        public Func<int, Memory<byte>> Enlarge;
+
+        static byte[] s_defaultNewline = new byte[] { (byte)'\n' };
+
+        public static BufferWriter Create(Span<byte> buffer) => new BufferWriter(buffer);
+        public static BufferWriter<TOutput> Create<TOutput>(TOutput output)
```

Comment:
nit: add new line between the two Create methods.

Reply:
will do